### DISCUSSION TITLE
Sending a User Message via External API

### DISCUSSION THREAD

**Discussion by Tournesol-me** [owncast/owncast#3993] [Q&A]
Hi, I'm trying to create a third party chat, and I would like to be able to send a message as a user.
I achieved to send one as a bot but the External API for User Message is marked deprecated in the api doc and I couldn't find why and how to post as a user then ?

Does anyone has any info on the way to go ?

**Comment by gabek** [comment]
A user and a bot are essentially the same. A bot is just a user where you can generate an access token.

I don't believe what you're trying to do, create a different chat interface for regular/non-bot users to use, would be possible. The websocket is used for that, and there's no external version available.

The deprecated API you saw was in use before chat used an actual account system, and using the API you could send a message using any display name you wanted. That is no longer possible.

I forget if you can use the `/integrations/chat/send` API to send a message as non-bot user as long as you have their access token. Have you tried that?
